src/toast/examples.js

```diff
@@ -124,11 +103,73 @@ class NotificationExample extends React.Component<{}, {toasts: []}> {
         >
           Negative toast
         </Button>
-        <Centered>
-          {this.state.toasts.map((toastOptions, index) => {
-            return <Toast key={index} {...toastOptions} />;
-          })}
-        </Centered>
+      </div>
+    );
+  }
+}
+
+class ToasterAdvancedExample extends React.Component<{}, {cleared: boolean}> {
+  keyToUpdate: React.Key;
+  state = {cleared: false};
+
+  componentDidMount() {
+    this.addNotifications();
+  }
+
+  addNotifications() {
+    toaster.show('Info notification');
+    toaster.show({
+      children: 'Positive notification',
+      kind: KIND.positive,
+    });
+    this.keyToUpdate = toaster.show({
+      children: 'Warning notification',
+      kind: KIND.warning,
+    });
+  }
+
+  render() {
+    return (
+      <div>
+        <Space>
+          <Button
+            onClick={() => {
+              const kind = {
+                '0': KIND.info,
+                '1': KIND.positive,
+                '2': KIND.warning,
```

Comment:
Have we considered how a user might want to use a custom toast regularly? Or should this not be part of the scope of Baseui?